html-formatter/ruby/assets/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.html
+*.js
+*.css
```

Comment:
Are these removed again when `make clean` is used?

Reply:
now they should be, thanks for pointing that out :)